libraries/lib-stretching-sequence/AudioSegmentFactory.cpp

```diff
@@ -38,14 +38,8 @@ AudioSegmentFactory::CreateAudioSegmentSequence(
 std::vector<std::shared_ptr<AudioSegment>>
 AudioSegmentFactory::CreateAudioSegmentSequenceForward(double t0) const
 {
-   ClipConstHolders sortedClips { mClips };
-   std::sort(
-      sortedClips.begin(), sortedClips.end(),
-      [](const ClipConstHolder& a, const ClipConstHolder& b) {
-         return a->GetPlayStartTime() < b->GetPlayStartTime();
-      });
    std::vector<std::shared_ptr<AudioSegment>> segments;
-   for (const auto& clip : sortedClips)
+   for (const auto& clip : mClips)
```

Comment:
Don’t add this commits!  I said I may strengthen the ordering guarantee in future.  That work is not done yet.